utils/joiner/joinpartition.cpp

```diff
@@ -821,7 +821,7 @@ uint64_t JoinPartition::writeByteStream(int which, ByteStream& bs)
     {
       fs.close();
       ostringstream os;
-      os << "Disk join could not write file " << filename << ": " << strerror(saveErrno) << endl;
+      os << "Disk join could not write to configured path check 'HashJoin' in columnstore.xml " << filename << ": " << strerror(saveErrno) << endl;
```

Comment:
We should be able to retrieve the actual path from ResourceManager instance here. I suggest to look at RM ctor.

Reply:
As per discussion, the filename path is actual and it is needed to retrieve the path from RM.